src/ripple/app/paths/PathRequest.cpp

```diff
@@ -241,12 +241,14 @@ bool PathRequest::isValid (RippleLineCache::ref crCache)
     If there's an error, we need to be sure to return it to the caller
     in all cases.
 */
-Json::Value PathRequest::doCreate (
+std::pair<bool, Json::Value>
+PathRequest::doCreate (
     RippleLineCache::ref& cache,
     Json::Value const& value,
     bool& valid)
 {
     Json::Value status;
+    bool valid;
```

Comment:
The parameter was supposed to go away. I must have mis-folded the branch. Will update.